src/_ert_forward_model_runner/runner.py

```diff
@@ -23,14 +28,42 @@ def __init__(self, jobs_data):
 
         self._set_environment()
 
+    def crate_manifest(self):
+        if not Path("manifest.json").exists():
+            return
+        with open("manifest.json", mode="r", encoding="utf-8") as f:
+            data = json.load(f)
```

Comment:
Its pretty weird that a function called `create_manifest` expects the file `manifest.json` to exist.